tests/savepoint/conftest.py

```diff
@@ -491,6 +491,17 @@ def failure_stride(pytestconfig):
     return int(pytestconfig.getoption("failure_stride"))
 
 
+@pytest.fixture()
+def print_domains(pytestconfig):
+    value = bool(pytestconfig.getoption("print_domains"))
+    original_value = fv3core.decorators.FrozenStencil.DEBUG
+    try:
+        fv3core.decorators.FrozenStencil.DEBUG = value
```

Comment:
If you are doing a FrozenStencil wide debug you might want to do and or on assignement to not override another future usage

Reply:
I think for now I’ll leave it as is, I had intended for it to override whatever value is there for the duration of the tests. Good to keep in mind if there’s ever a reason to enable this by default or through a different mechanism.

Reply:
I refactored the code to put this concern in the test code, which should avoid the issue you mention.